src/main/java/practice/StreamPractice.java

```diff
@@ -75,6 +112,11 @@ public List<String> getCatsNames(List<Person> peopleList, int femaleAge) {
      * parametrized with Candidate in CandidateValidator.
      */
     public List<String> validateCandidates(List<Candidate> candidates) {
-        return Collections.emptyList();
+        Predicate<Candidate> predicate = new CandidateValidator();
+        return candidates.stream()
+                .sorted((x, y) -> x.getName().compareTo(y.getName()))
```

Comment:
```suggestion
                .sorted(Comparator.comparing(Candidate::getName)))
```
same with method reference